.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Why did you add this in the repo? I don't think we do that for the other tree-sitter-xxx repos.

Reply:
it's a nice-to-have, and (with an additional ci/cd check) can enforce/check code styling/indentation/consistency, which, again, would be nice to have. I can remove this at your request

Reply:
I'm ok adding this, it's just that it should be in a different PR. We try to have a PR doing one thing and one thing only.